openqabot/approver.py

```diff
@@ -112,12 +135,16 @@ def get_jobs(self, job: JobAggr, api: str, inc: int) -> bool:
             if ok_job:
                 continue
             if self.is_job_marked_acceptable_for_incident(job, inc):
-                logger.info(
+                log.info(
                     "Ignoring failed job %s for incident %s due to openQA comment"
                     % (job.job_id, inc)
                 )
                 res["status"] = "passed"
             else:
+                log.info(
+                    "Found failed, not-ignored job %s for incident %s"
```

Comment:
This only logs the first failed job found.

Reply:
correct, only the first